api/envoy/api/v2/rds.proto

```diff
@@ -53,7 +61,10 @@ message RouteConfiguration {
   string name = 1;
 
   // An array of virtual hosts that make up the route table.
-  repeated route.VirtualHost virtual_hosts = 2 [(gogoproto.nullable) = false];
+  repeated route.VirtualHost virtual_hosts = 2; [(gogoproto.nullable) = false];
+  // An array of virtual hosts will be dynamically loaded via the VHDS API.
+  // [#not-implemented-hide:]
```

Comment:
Can you describe the intended final merge semantics?

Reply:
done.

Reply:
Smallest nit in the world: the last sentence is missing a period at the end.